toolchain/check/testdata/facet/convert_facet_value_value_to_generic_facet_value_value.carbon

```diff
@@ -29,6 +29,13 @@ class Goat {}
 impl Goat as Animal {}
 
 fn Feed[Food:! Edible, T:! Eats(Food)](e: T, food: Food) {}
+// CHECK:STDERR: convert_facet_value_value_to_generic_facet_value_value.carbon:[[@LINE+7]]:64: error: cannot convert type `T` that implements `Animal` into type implementing `Eats(Food)` [ConversionFailureFacetToFacet]
```

Comment:
Is this diagnostic expected? (If so I guess we need to rename this to `fail_todo_`)